addon/components/ember-dragula-container.js

```diff
@@ -3,10 +3,33 @@ import Ember from 'ember';
 const {Component, on} = Ember;
 
 export default Component.extend({
-  registerOnDrake: on('didInsertElement', function () {
+
+  getDrake: function(){
+    return this.get('parentView.drake') || this.get('drake');
+  },
+
+  didUpdateAttrs: function(attrs){
+    this._super(...arguments);
+    if (attrs.oldAttrs && attrs.oldAttrs.drake && attrs.oldAttrs.drake.containers) {
+      attrs.oldAttrs.drake.containers.removeObject(this.element);
+    }
+  },
+
+
+  didReceiveAttrs: function (attrs) {
+    this._super(...arguments);
     Ember.run.next(() => {
-      let drake = this.get('parentView').drake || this.drake;
-      drake.containers.push(this.element);
+      let drake = this.getDrake();
+      if (drake && !drake.containers.contains(this.element)) {
```

Comment:
Hey this doesnt seem to work at all.
"ember-dragula-container.js:26 Uncaught TypeError: drake.containers.contains is not a function"